package.json

```diff
@@ -198,7 +198,7 @@
     "@grpc/grpc-js": "^1.9.4",
     "@grpc/proto-loader": "^0.7.5",
     "@newrelic/ritm": "^7.2.0",
-    "@newrelic/security-agent": "^1.3.0",
+    "@newrelic/security-agent": "^1.5.0",
```

Comment:
this just updates the minimum version. due to semver ranges you'll get 1.5.0 on fresh installs. there isn't any that has to be done to get this working